WorkloadManagementSystem/Executor/InputData.py

```diff
@@ -313,8 +313,10 @@ def __getSiteCandidates( self, okReplicas, vo ):
       siteSet = set()
       for seName in replicas:
         result = self.__getSitesForSE( seName )
-        if result['OK']:
-          siteSet.update( result['Value'] )
+        self.jobLog.warn( "Could not get sites for SE %s: %s" % ( seName, result[ 'Message' ] ) )
```

Comment:
something wrong here. I do not see if and if result['OK'], there is no result['Message']